plugin/path_login.go

```diff
@@ -475,7 +487,7 @@ func (b *GcpAuthBackend) pathGceLogin(ctx context.Context, req *logical.Request,
 	auth := &logical.Auth{
 		InternalData: map[string]interface{}{},
 		Alias: &logical.Alias{
-			Name: fmt.Sprintf("gce-%s", strconv.FormatUint(instance.Id, 10)),
+			Name: alias,
```

Comment:
One interesting thing here is that, the alias _could_ be the `role_id`. However, the alias `metadata` fields don't change no matter what they select. Some of the fields would have a high rate of change if `role_id` were selected (`instance_name`, for example). This means if they selected `role_id`, they could have a performance hit.

@kalafut is this something we should merely document in this PR? Or should this PR (or a follow-on one) include the ability to configure metadata fields using the helper package being introduced in https://github.com/hashicorp/vault/pull/8783?

Reply:
Marking this as resolved because we chatted outside this PR, and decided that in a follow-on PR well also let them configure the metadata.